lib/models/builder.js

```diff
@@ -208,33 +227,41 @@ module.exports = CoreObject.extend({
     });
   },
 
+  _reportVizInfo: function (vizInfo) {
+    if (this.project && this.project.addons.length) {
+      this.project.addons.forEach(function (addon) {
+        if (addon._buildAnalytics) {
```

Comment:
lets just call this '__you_owe_stef_rob_and_dave_a_beer_or_something_if_you_invoke_thisBuildAnalytics`